### ISSUE TITLE
How to install kube-opex-analytics using helm3 (without tiller) 

### ISSUE THREAD

**Issue by Soma123** [rchakode/kube-opex-analytics#34]
I am trying to install kube-opex-analytics using helm3 (without tiller). Even though I did helm repo update couple of times but nothing seems to work. Do i need to have the repo "helm/kube-opex-analytics" locally downloaded to get this working.? Any help would be appreciated

**Comment by rchakode** [comment]
hi @Soma123,
we haven't yet validate kube-opex-analytics with Helm 3. We'll investigate the issue you have ASAP.
Meanwhile you may consider to generate the template and apply it with kubectl as below:

```
 helm template \
  --namespace kube-opex-analytics \
  --name kube-opex-analytics \
  helm/kube-opex-analytics/ | kubectl apply -f -
```

**Comment by Soma123** [comment]
> hi @Soma123,
> we haven't yet validate kube-opex-analytics with Helm 3. We'll investigate the issue you have ASAP.
> Meanwhile you may consider to generate the template and apply it with kubectl as below:
> 
> ```
>  helm template \
>   --namespace kube-opex-analytics \
>   --name kube-opex-analytics \
>   helm/kube-opex-analytics/ | kubectl apply -f -
> ```

If I am to use Helm template do i still need to have helm/kube-opex-analytics folder downloaded or checked in to the repository while applying it through an automated pipeline. Secondly where do i customize values like (namespace etc,)

**Comment by rchakode** [comment]
@Soma123, you can use any namespace you want. The one on the example is just for illustration.

About the need of downloading the helm folder, you should test it in your pipeline.

**Comment by rchakode** [comment]
hi @Soma123, we just released a new version of kube-opex-analytics ([v20.03.0](https://github.com/rchakode/kube-opex-analytics/releases/tag/v20.03.0)) including [documentation for Helm 3](https://github.com/rchakode/kube-opex-analytics#installation-using-helm-3-ie-without-tiller). 

let me know if you have any feedback.
